docs/schema_change_protocol.md

```diff
@@ -14,37 +14,38 @@ Author: Phil Snyder
 ## 2. Schema changes will be tested using a new stack:
```

Comment:
TODO: create a "template" for the study stack sceptre configuration. This can include a parameter for whether to include the trigger mentioned below (which involves a minor addition to the actual CFN template). Outside the scope of this PR, need to create JIRA issue.